Content.Shared/SurveillanceCamera/Components/SurveillanceCameraMapComponent.cs

```diff
@@ -0,0 +1,64 @@
+using System.Numerics;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.SurveillanceCamera.Components;
+
+/// <summary>
+/// Stores surveillance camera data for the camera map.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class SurveillanceCameraMapComponent : Component
+{
+    /// <summary>
+    /// Dictionary of cameras on on the current grid.
+    /// </summary>
+    [AutoNetworkedField]
```

Comment:
```suggestion
    [ViewVariables, AutoNetworkedField]
```
for debugging purposes